### ISSUE TITLE
Word in Context

### ISSUE THREAD

**Issue by Syavaprd** [AI21Labs/sense-bert#6]
Hello. Thank you very much for sharing your code. I'm a student and I want to reproduce your results on WiC competition. As I understood, in the inference you get two supersenses for the word in different contexts, after that you compare these two supersenses and decide if they are in the same meaning or not. The question is, do you finetune the model, if so, how is it done?
As I see it, you have to take supersense logits, after that compare them, may be with CosineSimilarity and get the loss.
